src/test/resources/3_0/security.yaml

```diff
@@ -61,3 +225,11 @@ components:
       type: http
       scheme: bearer
       bearerFormat: JWT
+    OAuth2PasswordBearer:
+      type: oauth2
+      flows:
+        password:
+          scopes:
+            me: Read information about the current user.
+            items: Read items.
```

Comment:
Items is vague and has no intrinsic meaning. How about something like `userEmails`, 'userAddresses`?

Reply:
Fixed. Used petstore items.